src/server/utils/logging/production-formatter.js

```diff
@@ -86,6 +86,10 @@ function leadingErrorWithOtherArgs(level, ...args) {
     };
   }
 
+  if (err.name === 'ServerSideReportedError') {
```

Comment:
is this dependent on the instance of the Error or can duck typing(?) be used instead? _maybe_
```suggestion
  if (err.metaData) {
```

Reply:
This will duplicate the metaData for the clientReported error `err.request.metaData and err.metaData`